cmsplugin_articles_ai/models/articles.py

```diff
@@ -106,6 +114,10 @@ class Article(models.Model):
         verbose_name=_("URL slug"),
         db_index=True,
     )
+    language = LanguageField(
```

Comment:
CMS plugins have "langauge_filter" that is empty by default. And the list view is language agnostic by default, but accepts language filter parameter as GET parameter. So by default every available view is language agnostic.